File: src/tests/Markify.Solution.Tests/CSharp.Project/Wellformed/TestClass.cs
namespace CSharp.Project.Wellformed
{
    public class TestClass
    {
        protected class InnerTestClass
        {
        }
    }

    public sealed class TestClass1
    {
    }

    internal abstract class TestClass2
    {
    }

    public sealed partial class TestClass3
    {
    }

    public sealed partial class TestClass3
    {
    }

    public abstract partial class TestClass5
    {
    }

    public abstract partial class TestClass5
    {
    }

    internal static class TestClass7
    {
    }
}

File: src/tests/Markify.Solution.Tests/VisualBasic.Project/Wellformed/TestClass.vb
Namespace CSharp.Project.Wellformed
    Public Class TestClass
        Protected Class InnerTestClass
        End Class 
    End Class 

    Public NotInheritable Class TestClass1
    End Class 

    Friend MustInherit Class TestClass2
    End Class 

    Public NotInheritable Partial class TestClass3
    End Class 

    Public NotInheritable Partial class TestClass3
    End Class 

    Public MustInherit Partial class TestClass5
    End Class 

    Public MustInherit Partial class TestClass5
    End Class 

    Public Module TestClass7
    End Module
End Namespace